src/components/ObsDetails/ObsDetails.js

```diff
@@ -159,6 +165,25 @@ const ObsDetails = (): Node => {
     } );
   };
 
+  const createIdentificationMutation = useAuthenticatedMutation(
+    ( idParams, optsWithAuth ) => createIdentification( idParams, optsWithAuth ),
+    {
+      onSuccess: data => setIds( [...ids, data[0]] ),
+      onError: e => {
+        let error = null;
+        if ( e ) {
+          error = t( "Couldnt-create-identification", { error: e.message } );
```

Comment:
This key doesn't accept a variable, so you either need to alter it or make a different key.